trin-beacon/src/storage.rs

```diff
@@ -301,11 +299,19 @@ impl BeaconStorage {
     fn db_insert(
         &self,
         content_id: &[u8; 32],
-        content_key: &String,
+        content_key: &Vec<u8>,
         value: &Vec<u8>,
     ) -> Result<usize, ContentStoreError> {
         let conn = self.sql_connection_pool.get()?;
-        insert_value(conn, content_id, content_key, value, u8::from(self.network))
+        Ok(conn.execute(
+            INSERT_QUERY_BEACON,
+            params![
+                content_id.to_vec(),
```

Comment:
The `content_id.to_vec()` will make a copy and I don't think it's necessary.

If you can't pass `content_id` directly, you should be able to pass: `content_id.as_slice()`.

Same in many other places as well.